docs/next.config.js

```diff
@@ -154,4 +154,10 @@ module.exports = {
       { source: '/api/:rest*', destination: '/api-docs/:rest*' },
     ];
   },
+  redirects:
+    process.env.NODE_ENV !== 'production'
+      ? async () => {
+          return [{ source: `/:lang(${LANGUAGES.join('|')})?/:rest*`, destination: '/:rest*' }];
```

Comment:
The content here seems off topic, did you intent to redirect to the data grid component?

Reply:
:roll_eyes: I commited one of my tests copied from the core...
It should be the same redirection as in the Netlify config but for the local env